docs/data/base/components/checkbox-group/UnstyledCheckboxGroupIntroduction/css/index.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from 'react';
+import { useTheme } from '@mui/system';
+import * as Checkbox from '@base_ui/react/Checkbox';
+import * as CheckboxGroup from '@base_ui/react/CheckboxGroup';
+import * as Field from '@base_ui/react/Field';
+import Check from '@mui/icons-material/Check';
+
+export default function UnstyledCheckboxIndeterminateGroup() {
+  return (
+    <Field.Root>
+      <div style={{ display: 'flex', flexDirection: 'column' }}>
+        <CheckboxGroup.Root defaultValue={['red']}>
+          <Field.Label className="CheckboxGroup-label">Colors</Field.Label>
+          <div style={{ display: 'flex', flexDirection: 'column' }}>
+            <Label className="Checkbox-label">
+              <Checkbox.Root className="Checkbox" name="red">
+                <Checkbox.Indicator className="Checkbox-indicator">
+                  <Check className="Check" />
+                </Checkbox.Indicator>
+              </Checkbox.Root>
+              Red
+            </Label>
+            <Label className="Checkbox-label">
+              <Checkbox.Root className="Checkbox" name="green">
+                <Checkbox.Indicator className="Checkbox-indicator">
+                  <Check className="Check" />
+                </Checkbox.Indicator>
+              </Checkbox.Root>
+              Green
+            </Label>
+            <Label className="Checkbox-label">
+              <Checkbox.Root className="Checkbox" name="blue">
+                <Checkbox.Indicator className="Checkbox-indicator">
+                  <Check className="Check" />
+                </Checkbox.Indicator>
+              </Checkbox.Root>
+              Blue
+            </Label>
+          </div>
+        </CheckboxGroup.Root>
+        <Styles />
+      </div>
+    </Field.Root>
+  );
+}
+
+const grey = {
+  100: '#E5EAF2',
+  300: '#C7D0DD',
+  500: '#9DA8B7',
+  600: '#6B7A90',
+  800: '#303740',
+  900: '#1C2025',
+};
+
+function useIsDarkMode() {
```

Comment:
Define the dark mode styles by targeting the `body.mode-dark` class, please. It'll make it easier to extract the styles to a separate CSS file in the future.